app/src/main/java/com/mikepenz/unsplash/views/adapters/ImageAdapter.java

```diff
@@ -76,6 +76,7 @@ public ImagesViewHolder onCreateViewHolder(ViewGroup viewGroup, int position) {
     public void onBindViewHolder(final ImagesViewHolder imagesViewHolder, final int position) {
 
         final Image currentImage = mImages.get(position);
+        imagesViewHolder.image.setImageBitmap(null);
```

Comment:
there's no "image" in the holder
-> should be: "imagesViewHolder.imageView.setImageBitmap(null);"
also please add the source and the note about this one as comment above the line. 

Also please to the .setImageBitmap under the setDrawingCache line